specs/asgi.rst

```diff
@@ -232,9 +232,9 @@ defined by the protocol spec. Examples of a message ``type`` value include
 .. note::
 
   Applications should actively reject any protocol that they do not understand
-  with an Exception (of any type). Failure to do this may result in the server
-  thinking you support a protocol you don't, which can be confusing with
-  the Lifespan protocol, as the server will wait to start until you tell it.
+  with an exception (of any type). Failure to do this may result in the server
+  thinking you support a protocol you don't, which can complicate the Lifespan
```

Comment:
I don't like complicate as a substitute for confusing; try rewording it? Needs to convey that this is not apparent to the end-user, and potentially dangerous.

Reply:
Makes sense! I think I might just back this change out given that additional context.